client/components/LearnerViewDirective.js

```diff
@@ -284,6 +368,14 @@ tie.directive('learnerView', [function() {
           }
         };
 
+        $scope.navigateToQuestion = function(index) {
+          if ($scope.unlockedQuestions[index]) {
+            $scope.currentQuestionIndex = index;
+            var questionId = $scope.questionIds[$scope.currentQuestionIndex];
+            loadQuestion(questionId, questionSet.getIntroductionParagraphs());
```

Comment:
We may need an intro for each exercise because when you navigate between questions, the intro text for the first exercise may or may not apply to the subsequent questions. At a minimum, use a generic intro for each exercise. My gut feeling is that a unqiue intro for each exercise will make it seem more conversational. Right now, without an intro for each exercise, navigating between exercises results in neither, which is not a good UX.